coderd/users_test.go

```diff
@@ -1209,7 +1209,7 @@ func TestPaginatedUsers(t *testing.T) {
 			t.Parallel()
 
 			// This test takes longer than a long time.
-			ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong*2)
+			ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong*3)
```

Comment:
why did we increase timeout if we decreased RBAC time?

Reply:
I am unsure why this test was timing out. I can spend more time and look into it

Reply:
This test takes <3s on my local machine. I have no clue why it was timing out... going to drop back to *2 and see what I get.